apps/frontend/src/modules/users/slices/users.ts

```diff
@@ -1,9 +1,10 @@
-import { loadAll } from "./actions.js";
+import { loadAll, updateUser } from "./actions.js";
 import { actions } from "./users.slice.js";
 
 const allActions = {
 	...actions,
 	loadAll,
+	updateUser,
```

Comment:
`updateUser` action will be used like `usersActions.updateUser(...)`, so it's better to rename it to `update`, so we do not violate A5 of QC

Reply:
Fair enough